### ISSUE TITLE
Rage broken

### ISSUE THREAD

**Issue by itteerde** [itteerde/fvttconfig#2]
does not work...

effect `@scale.barbarian.rage-damage` ?

**Comment by itteerde** [comment]
fix by editing the Barbarian Advancement adding
`@scale.barbarian.rage-damage` to the Actor by duplicating and editing the existing wrong one from ddb-importer.
Also the existing one is completely wrong and should be either the new one or `@scale.barbarian.rages`, not `@scale.barbarian.rage`.

**Comment by itteerde** [comment]
https://foundryvtt.wiki/en/development/guides/active-effects

might also require more of the following:
![Screenshot 2022-10-13 203919](https://user-images.githubusercontent.com/8367477/195679507-7a6753ff-9134-43cd-a10b-9ec5d1f40844.png)

**Comment by itteerde** [comment]
Added manual fixing to [afterDDBImport.md](../Spelljammer/sop/afterDDBImport.md) SOP for Spelljammer. This should become a macro for as long as it is not dealt with by DDB-import or DFreds (think it never will be) or we make it work according to the claims by MRPrimate who claims that it works for him with MidiQOL and DAE.

**Comment by itteerde** [comment]
trying to make the macro work (and in the progress work out how to edit stuff by macro), but so far [the macro](https://github.com/itteerde/fvttconfig/blob/main/Spelljammer/config/macros/ddb-import_post_hildr.js) does not work.

**Comment by itteerde** [comment]
Can fix it with a fixed class item but that triggers tha advancement when being dropped -- still a bit faster than editing (~20sec). I should be able to update the fix part on the class item created by ddb-importer by macro.

**Comment by itteerde** [comment]
every solution editing `scale` values need to be aware of Actor

```
 /**
   * Derive any values that have been scaled by the Advancement system.
   * Mutates the value of the `system.scale` object.
   * @protected
   */
  _prepareScaleValues() {
    this.system.scale = Object.entries(this.classes).reduce((scale, [identifier, cls]) => {
      scale[identifier] = cls.scaleValues;
      if ( cls.subclass ) scale[cls.subclass.identifier] = cls.subclass.scaleValues;
      return scale;
    }, {});
  }
```

**Comment by itteerde** [comment]
generally there is
```
const itemData = {type: "loot", name: "Steve"};
await actor.createEmbeddedDocuments("Item", [{itemData}]);
```

more concrete we have

```
const newScaleValue = {
  type: "ScaleValue",
  configuration: {
    identifier: "rage-damage",
    type: "numeric",
    scale: { "1": { value: 2 }, "9": { value: 3 }, "16": { value: 4 } }
  },
  title: "Rage Damage"
}
await myClassItem.createAdvancement("ScaleValue", newScaleValue);
```

[Zhell's comment](https://discord.com/channels/170995199584108546/699750150674972743/1030829098873921567)

**Comment by itteerde** [comment]
MRPrimate:
> I have "fixed" this in v3.1.16, it will now add the SRD advancements where there is no identifier clash.

indeed, works.
